images/smarter-device-manager/config/latest.apko.yaml

```diff
@@ -0,0 +1,19 @@
+contents:
+  packages:
+    # smarter-device-manager defined in .main.tf
+
+accounts:
+  groups:
+    - groupname: nonroot
+      gid: 65532
+  users:
+    - username: nonroot
+      uid: 65532
+      gid: 65532
+  # smarter-device-manager requires root to interact with directories creaed by device plugins
+  run-as: 0
+
+entrypoint:
+  command: /usr/bin/smarter-device-management
+
+cmd: -config=/etc/smarter-device-manager/conf.yaml -logtostderr=true -v=2
```

Comment:
nit - I wonder if the log level can be made configureable?  I.e. via an env var or mounted config?  Just thinking that folks may want the ability to change this for production or when debugging?

Reply:
Sure I can take a look. with this particular application though it'll simply exit and log nothing if there is an issue, i.e:

```
# smarter-device-management -config=/etc/smarter-device-manager/conf.yaml
#
```

Nothing informational either so not the easiest to debug. Kind of my thinking for enabling this and it didn't look to verbose. That said I can certainly look to put behind a flag and document if preferred

Reply:
ok well let's not block on this, see what you can find.

Reply:
Also meant to add thats what the vendor currently does in their image, so at least we're consistent in a sense:
- https://gitlab.com/arm-research/smarter/smarter-device-manager/-/blob/master/Dockerfile#L22